admin/admin-filters-post.php

```diff
@@ -211,7 +167,7 @@ public function bulk_edit_posts() {
 			if ( $lang = $this->model->get_language( sanitize_key( $_GET['inline_lang_choice'] ) ) ) {
 				$post_ids = array_map( 'intval', (array) $_REQUEST['post'] );
 				foreach ( $post_ids as $post_id ) {
-					$this->inline_save_language( $post_id, $lang );
+					$this->model->post->update_language( $post_id, $lang, get_post_type( $post_id ) );
```

Comment:
You kept the old version. I suggest to restore the method `inline_save_language()`, keep the validation in this function and call the new method `update_language()` with 2 parameters there.